src/pages/ReviewResults.js

```diff
@@ -0,0 +1,107 @@
+    this.getData(darId);
+  }
+
+  async getData(darId) {
+
+    //get information on datasets, consent, researcher, and access request
+    const getDarData = async (darId) => {
+      let datasets;
+      let darInfo;
+      let consent;
+      let researcherProfile;
+
+      try {
+        darInfo = await DAR.getPartialDarRequest(darId);
+        const researcherPromise = await Researcher.getResearcherProfile(darInfo.userId);
+        const datasetsPromise = darInfo.datasetIds.map((id) => {
+          return DataSet.getDataSetsByDatasetId(id);
+        });
+        const consentPromise = await DAR.getDarConsent(darId);
+        [consent, datasets, researcherProfile] = await Promise.all([
+          consentPromise,
+          Promise.all(datasetsPromise),
+          researcherPromise
+        ]);
+
+      } catch (error) {
+        Notifications.showError({text: 'Error initializing DAR Data'});
+        return Promise.reject(error);
+      }
+
+      return {datasets, darInfo, consent, researcherProfile};
+    };
+
+
+    //get information on access election and research purpose election
+    const getElectionInformation = async (darId) => {
+      let accessElection;
+      let accessElectionReview;
+      let rpElectionReview;
+
+      try {
+        accessElection = await Election.findElectionByDarId(darId);
+      } catch (error) {
+        //access election is null
+        Notifications.showInformation({text: 'There is not an active data access election for this DAR.'});
+      }
+      try {
+        accessElectionReview = fp.isNil(accessElection) ? null : await Election.findDataAccessElectionReview(accessElection.electionId, true);
+        rpElectionReview = fp.isNil(accessElection) ? null : await Election.findRPElectionReview(accessElection.electionId, false);
+      } catch (error) {
+        Notifications.showError({text: 'Error initializing Election Data'});
+        return Promise.reject(error);
+      }
+      return { accessElection, accessElectionReview, rpElectionReview };
+    };
+
+
+    const [electionData, darData] = await Promise.all([
+      getElectionInformation(darId),
+      getDarData(darId)
+    ]);
+
+    const {datasets, darInfo, consent, researcherProfile} = darData;
+    const {accessElectionReview, accessElection, rpElectionReview} = electionData;
+
+    this.setState({ darId, accessElection, darInfo, consent, accessElectionReview, rpElectionReview, researcherProfile, datasets });
+  }
+
+    render() {
+     if (!fp.isNil(this.state)) {
+      const voteAsChair = true;
+      const {history, match} = this.props;
+      const {darId, darInfo, datasets, consent, researcherProfile, accessElection, accessElectionReview, rpElectionReview } = this.state;
+
+      return div({isRendered: !fp.isNil(this.state), id: 'container', style: {margin: '2rem'}}, [
+        div({id: 'header', style: {...SECTION, padding: '1rem 0'}}, [
+          AccessReviewHeader({history, match})
```

Comment:
`AccessReviewHeader` doesn't seem to use the history prop - that may have been something we deleted at some point and was never removed from existing calls.

Reply:
Ah ... I see it now, you removed it when moving to the constructor approach. Will comment there.